.github/workflows/gitlab-ci.yaml

```
@@ -78,4 +78,12 @@
       - name: Run tests
         run: |
-          python manage.py test
+          python manage.py test
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+
+      - name: Run Go script
+        run: go run path/to/your_script.go
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)